projects/plugins/search/tests/e2e/README.md

```diff
@@ -0,0 +1,132 @@
+[![Reports status](https://img.shields.io/website?down_color=grey&down_message=Dashboard%20offline&style=for-the-badge&label=E2E%20TEST%20REPORTS&up_color=green&up_message=see%20dashboard&url=https%3A%2F%2Fautomattic.github.io%2Fjetpack-e2e-reports%2F%23%2F)](https://automattic.github.io/jetpack-e2e-reports)
```

Comment:
Just wondering, how useful it is to have similar README files for every plugin that has it's own e2e suite. There are definitely benefits (such as how to run this particular suite), but I'm not sure about the technical bits: docker, tunneling, config, etc. Mainly, I am concerned about the need to maintain these similar docs.

Reply:
Yeah, I agree we can optimize the doc files further. I will address this later on.